docs/std-eval.md

```diff
@@ -1,12 +1,18 @@
-# strconv library
+# eval
 
-`eval` has one function which is `value` which takes a string respresenting an
-expression and returns an arrai value.
+The `eval` contains functions which converts raw string into a built-in arrai values.
+
+## `eval.value(s <: string) <: any`
```

Comment:
Sometimes you include the full context. Other times you only present the name. I think we should consistently show the full context, including the `//`, because that's typically how these functions will be used: `//eval.value(...)`.

Reply:
yep some of them are modifications of existing docs, I'll make it more consistent